### ISSUE TITLE
Occasional double or triple writes for same data

### ISSUE THREAD

**Issue by hirochri** [influxdata/influxdb#7023]
Ran into a similar issue as mentioned in [3315](https://github.com/influxdata/influxdb/issues/3315), [6946](https://github.com/influxdata/influxdb/issues/6946), and [6853](https://github.com/influxdata/influxdb/issues/6853).

Basically, when using curl to post data from a bunch of text files, occasionally some points will be written into the database 2 or 3 times, with the points being exactly the same (measurement, tags, fields, everything). I am in parallel pushing the same data into a different database using UDP, and here I don't have the duplication issue. When I run `service influxd restart`, it appears to have gotten rid of the duplicates once the system comes back up.

This happens with influxdb version 0.13.0 on Ubuntu 16.04

I'll happily provide more information if needed, but just wanted to call attention to the issue again!

**Comment by jwilder** [comment]
Can you test with a current nightly release?

**Comment by jwilder** [comment]
#7027 should fix this.  Sounds very similar to #6946

**Comment by jwilder** [comment]
Duplicate of #6946

**Comment by hirochri** [comment]
Sorry I was unable to test with a nightly release quickly, and #7027 looks good, thanks!
